go/libkb/pgp_key.go

```diff
@@ -346,7 +347,44 @@ func cleanPGPInput(s string) string {
 // note:  openpgp.ReadArmoredKeyRing only returns the first block.
 // It will never return multiple entities.
 func ReadOneKeyFromString(originalArmor string) (*PGPKeyBundle, *Warnings, error) {
+	return readOneKeyFromString(originalArmor /* liberal */, false)
```

Comment:
I'm nitpicking, but grep through client repo shows that block comments with argument name went after the boolean.